include/siminusminus/containers/inmutablestring.hpp

```diff
@@ -28,11 +31,115 @@ namespace containers
 class InmutableString
```

Comment:
I don't know what you refer, there are some information about inmutable string just above of that line (You wrote that when you give me the hpp file). ¿You want that I rewrite something?